lib/code_corps/model/github_pull_request.ex

```diff
@@ -73,4 +74,14 @@ defmodule CodeCorps.GithubPullRequest do
     struct
     |> changeset(params)
   end
+
+  defp add_merged(%Ecto.Changeset{changes: %{merged: merged}} = changeset) when is_boolean(merged) do
+    changeset
+  end
+  defp add_merged(%Ecto.Changeset{changes: %{merged_at: _}} = changeset) do
+    changeset |> put_change(:merged, true)
+  end
+  defp add_merged(%Ecto.Changeset{} = changeset) do
+    changeset |> put_change(:merged, false)
+  end
```

Comment:
Apparently GitHub does not pass `merged` along when listing the records.

Can also remove:

```
additions changed_files comments commits deletions mergeable_state review_comments 
```